src/core/components/InstallButton/index.js

```diff
@@ -72,6 +84,29 @@ export class InstallButtonBase extends React.Component {
           {i18n.gettext('Install Theme')}
         </Button>
       );
+    } else if (addon.type === ADDON_TYPE_OPENSEARCH) {
+      const onClick = (event) => {
+        event.preventDefault();
+        event.stopPropagation();
+
+        if (compatible) {
```

Comment:
It looks like this if statement is not necessary since the button will be disabled. If so, I think you should remove the if statement because at least we'd see an error in the console if `onClick` was triggered by a programming mistake -- this would help us track down the bug.